docs/conf.py

```diff
@@ -150,6 +150,7 @@ def doc_redirect_include(file_path):
     "https://github.com/Ouranosinc/cowbird/*",    # limit only Cowbird so others are still checked
     # ignore private links
     "https://github.com/Ouranosinc/PAVICS/*",
+    "https://github.com/bird-house/birdhouse-deploy/*",
```

Comment:
See if  https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_anchors_ignore works instead. It would be a better ignore condition.

Reply:
It works but it only uses the anchor part, so I have to put the failing links specific anchors. I updated the code accordingly.

Note : I saw there was also the [linkcheck_anchors_ignore_for_url](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_anchors_ignore_for_url) option, but the `sphinx-build` version on the project is not recent enough to have this feature.

Reply:
Ok. You can try using it with a more recent version of Sphinx and see if it works. I think the `<7` requirement was only set to avoid flaky build because v7 was recent at that time. Using 7.1 is probably ok. If the build fails or causes more trouble that it is worth, then your current edit is fine as well.